<launch>
  <include file="$(find bitbots_ros_control)/launch/ros_control.launch" />
  <include file="$(find bitbots_bringup)/launch/load_robot_description.launch" />
  <node name="dynamic_kick" pkg="bitbots_dynamic_kick" type="KickNode" output="screen">
    <rosparam command="load" file="$(find bitbots_dynamic_kick)/config/kick_config.yaml" />
    <remap from="/kick_motor_goals" to="/DynamixelController/command"/>
  </node>
</launch>
